webapp/src/blocks/card.ts

```diff
@@ -8,7 +8,7 @@ interface Card extends IBlock {
     readonly icon: string
     readonly isTemplate: boolean
     readonly properties: Readonly<Record<string, string | string[]>>
-    readonly contentOrder: readonly string[]
+    readonly contentOrder: readonly (string|string[])[]
```

Comment:
I guess this is because backward compatibility, but this type looks weird.

Maybe would be simpler if we write that as `string[]|string[][]`

Reply:
Done!